include/zephyr/dt-bindings/misc/nxp-inputmux-rt6xx.h

```diff
@@ -4,37 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#ifndef __NXP_ADSP_INPUTMUX_INTC_RT6XX_H__
-#define __NXP_ADSP_INPUTMUX_INTC_RT6XX_H__
-
-#define ADSP_IRQ_5 5
```

Comment:
Could you find a way to unify this block of changes with previous commits? This is being introduced a few commits back but then are removed here, all in the same PR.

Reply:
I redid the whole commit history, so no commits without comments and subsequent commits are there.